smithy-smoke-test-traits/src/test/resources/software/amazon/smithy/smoketests/traits/errorfiles/vendor-params-validation.errors

```diff
@@ -0,0 +1,4 @@
+[WARNING] smithy.example#SayHello: Smoke test case defined a `vendorParamsShape` but no `vendorParams` | SmokeTestCase
+[WARNING] smithy.example#SayHello: smithy.test#smokeTests.1.vendorParams: Invalid structure member `bar` found for `smithy.example#VendorParams` | SmokeTestCase
```

Comment:
Is there any way to replace "1" here with the case ID?

Reply:
Yes, done. There's probably other places we can do the same